temporal/t.rast.colors/t.rast.colors.py

```diff
@@ -148,13 +149,7 @@ def main():
     if rows:
         # Create the r.colors input file
         filename = gs.tempfile(True)
```

Comment:
gs.tempfile should be probably replaced with standard python NamedTemporaryFile and similarly in the other case